### ISSUE TITLE
The availability of a quick fix in the terminal should be announced to screen readers

### ISSUE THREAD

**Issue by connor4312** [microsoft/vscode#167593]
Testing #166761 on the Ubuntu built-in screen reader

1. Have the screen reader on
2. Enter a command that triggers a quick fix
3. Notice there is no screen reader announcement for it.

We don't have an announcement in the editor, but I think it makes sense there, since quick fixes can be available when just navigating around code, which would be noisy (literally). However, in a terminal an action is available as a direct result of executing something, so I think announcing it would make sense.

**Comment by Tyriar** [comment]
There's an audio cue which I think covers this already?

**Comment by connor4312** [comment]
Ah, didn't notice that. Perfect 🙂
